apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/exception/OrganizationAlreadyExistsExceptionTest.java

```diff
@@ -0,0 +1,20 @@
+package org.sagebionetworks.openchallenges.organization.service.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class OrganizationAlreadyExistsExceptionTest {
+
+  @Test
+  public void OrganizationAlreadyExistsException_ShouldReturnDetail_WhenDetailKeyIsPassed() {
+    // Set up the input detail
+    String detail = "Errordetail";
+
+    // Create an instance of OrganizationAlreadyExistsException
+    OrganizationAlreadyExistsException exception = new OrganizationAlreadyExistsException(detail);
```

Comment:
What you call `detail` here is defined as `message` in the implementation of `OrganizationAlreadyExistsException`. You should use `message` instead of creating a new name for this variable.

Reply:
This comment applies to other files in this PR.

Reply:
> What you call `detail` here is defined as `message` in the implementation of `OrganizationAlreadyExistsException`. You should use `message` instead of creating a new name for this variable.

![image](https://github.com/Sage-Bionetworks/sage-monorepo/assets/122999770/d11602f5-a5c9-4354-a3eb-09164ec5fac8)
I don't see where detail is called message in OrganizationAlreadyExistsException.java
Are you sure I should have changed detail to message?

Reply:
> @tschaffter
> 
> > What you call `detail` here is defined as `message` in the implementation of `OrganizationAlreadyExistsException`. You should use `message` instead of creating a new name for this variable.
> 
> ![image](https://private-user-images.githubusercontent.com/122999770/284729441-d11602f5-a5c9-4354-a3eb-09164ec5fac8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEyMTM3NzcsIm5iZiI6MTcwMTIxMzQ3NywicGF0aCI6Ii8xMjI5OTk3NzAvMjg0NzI5NDQxLWQxMTYwMmY1LWE1YzktNDM1NC1hM2ViLTA5MTY0ZWM1ZmFjOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMTI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTEyOFQyMzE3NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZDhkMWM5ZDM2YmM3ZTM5NTdhNmM2MTlmYzQyYjM0ZmMzNTVkODgyNjQ1NmFiZGIwMWY3MjZlZmE1ZTY0ZGJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zD7QqX5I3u9o8fjn1WhtyK3zV1N0RB54g-nVXcwuGQc) I don't see where detail is called message in OrganizationAlreadyExistsException.java Are you sure I should have changed detail to message?

@tschaffter I was wondering if you could provide feedback on this question.

Reply:
@mdsage1 Never mind, I'm the one mistaking. Marking this issue as resolved.